api-conversion/src/main/java/io/strimzi/kafka/api/conversion/converter/Conversion.java

```diff
@@ -252,6 +266,35 @@ public void convert(T from) {
                 }
             }
 
+            private Object getValueFromPath(T from, String path, PropertyUtils pu)   {
+                List<String> pathNames = pathTokens(path);
+                Object target = from;
+
+                for (int i = 0; i < pathNames.size(); i++) {
+                    Class<?> targetClass = target.getClass();
+                    String name = pathNames.get(i);
+                    Property property = pu.getProperty(targetClass, name);
+
+                    if (property == null || !property.isReadable()) {
+                        return null;
+                    }
+
+                    Object value = property.get(target);
+
+                    if (value == null) {
+                        return null;
```

Comment:
Is it necessary to distinguish the cases of a path whose final property is null from the case where the complete path didn't exist?

Reply:
No, that should not be necessary.